sqlmesh/core/renderer.py

```diff
@@ -310,10 +314,12 @@ def __init__(
         macro_definitions: t.List[d.MacroDef],
         schema: t.Optional[t.Dict[str, t.Any]] = None,
         model_name: t.Optional[str] = None,
+        model_fqn: t.Optional[str] = None,
```

Comment:
Do we need both `model_name` and `model_fqn`? Seems rather redundant. Given that rendered has access to default catalog, can't we just normalize in place? Or get rid of `model_name`?

Reply:
If we just passed in `model_name` then we can get the `model_fqn` because `model_name` has already been normalized and therefore to create the fqn we would do a double normalization.

FQN alone cannot be used to determine name, even if you know default catalog, since the user could have given the model a fully qualified name even though the catalog they used in the name matches what the default catalog is. 

It looks like `model_name` is just used logging warning/errors so if we were ok with logging fqn then we could drop name.

Reply:
>It looks like model_name is just used logging warning/errors so if we were ok with logging fqn then we could drop name.

I think it's ok to log fqn